cylc/flow/task_remote_mgr.py

```diff
@@ -259,7 +259,7 @@ def remote_tidy(self):
                 proc.terminate()
             except OSError:
                 pass
-            out, err = proc.communicate()
+            out, err = (f.decode() for f in proc.communicate())
```

Comment:
Minor bug that made a mess of reported error but didn't cause traceback in the Scheduler.

Reply:
Huh, why do we need string `decode()` here?

Reply:
To answer myself:
> `communicate()` returns a tuple (stdout_data, stderr_data). The data will be strings if streams were opened in text mode; otherwise, bytes.

Should we just be specifying text mode (or explicit encoding) instead, for clarity?

Reply:
I've only just learned about "text" mode myself recently, looks good, presumably it just does the decode for you using UTF-8.